library/src/test/java/com/paypal/messages/config/message/PayPalMessageConfigTest.kt

```diff
@@ -25,31 +26,34 @@ class PayPalMessageConfigTest {
 		assertEquals(config.eventsCallbacks, eventsCallbacks)
 	}
 
-// 	@Test
+	@Test
 	fun testSetGlobalAnalytics() {
 		val name = "integration_name"
 		val version = "integration_version"
+		val deviceId = "device_id"
+		val sessionId = "session_id"
+
+		assertEquals("", GlobalAnalytics.integrationName)
+		assertEquals("", GlobalAnalytics.integrationVersion)
+		assertEquals("", GlobalAnalytics.deviceId)
+		assertEquals("", GlobalAnalytics.sessionId)
```

Comment:
These asserts are also in GlobalAnalyticsTest.kt. We should be safe to remove 1 copy